float32[3] tri